editor/plugins/path_2d_editor_plugin.cpp

```diff
@@ -298,6 +303,26 @@ bool Path2DEditor::forward_gui_input(const Ref<InputEvent> &p_event) {
 	Ref<InputEventMouseMotion> mm = p_event;
 
 	if (mm.is_valid()) {
+		// When both control points were in range of click,
+		// pick the point that drags the curve outwards.
+		if (control_points_in_range == 2) {
+			control_points_in_range = 0;
+			Ref<Curve2D> curve = node->get_curve();
+			Point2 relative = mm->get_relative();
```

Comment:
Mouse motion should to be transformed to be relative to the edited Path2D (and thus its curve), it can be rotated etc.

Assumming the transforming in the existing code down below is correct:
```suggestion
			Transform2D xform = canvas_item_editor->get_canvas_transform() * node->get_screen_transform();
			Point2 relative = xform.affine_inverse().basis_xform(mm->get_relative());
```